openhcl/underhill_core/src/worker.rs

```diff
@@ -1719,11 +1719,15 @@ async fn new_underhill_vm(
         ))
     };
 
+    // ARM64 always bounces, as the OpenHCL kernel does not
+    // have access to VTL0 pages. Necessary until #273 is resolved.
+    let always_bounce = cfg!(guest_arch = "aarch64");
```

Comment:
em, why we need set always_bounce here instead of blockdevice? Because it's only affected on OpenHCL?